contrib/inventory/ec2.py

```diff
@@ -1144,6 +1145,10 @@ def get_host_info_dict_from_instance(self, instance):
                     group_names.append(group.name)
                 instance_vars["ec2_security_group_ids"] = ','.join([str(i) for i in group_ids])
                 instance_vars["ec2_security_group_names"] = ','.join([str(i) for i in group_names])
+            elif key == 'ec2_block_device_mapping':
+                instance_vars["ec2_block_devices"] = {}
+                for k, v in value.iteritems():
```

Comment:
For python3 compat, avoid iteritems unless you're dealing with large data sets.  If you are, then use ansible.compat.six.iteritems.